packages/client/hmi-client/src/components/model/tera-model.vue

```diff
@@ -149,12 +149,13 @@ const optionsMenuPt = {
 
 async function updateModelContent(updatedModel: Model) {
 	if (!useProjects().hasEditPermission()) {
+		logger.error('You do not have permission to edit this model.');
```

Comment:
I would prefer for the button or actions allowing the users to edit the model to be disabled if they do not have the rights to do so.

Reply:
I'll write a comment about that